cmd/virt-handler/virt-handler.go

```diff
@@ -223,9 +295,15 @@ func (app *virtHandlerApp) AddFlags() {
 	flag.StringVar(&app.PodIpAddress, "pod-ip-address", podIpAddress,
 		"The pod ip address")
 
+	flag.StringVar(&app.PodName, "pod-name", hostOverride,
+		"The pod name")
+
 	flag.StringVar(&app.VirtShareDir, "kubevirt-share-dir", virtShareDir,
 		"Shared directory between virt-handler and virt-launcher")
 
+	flag.StringVar(&app.CertDir, "cert-dir", certificateDir,
```

Comment:
is there a reason this needs to be configurable?

Reply:
I was thinking about keeping the certificates between container crashes, so taht we don't create ne CRS in that scenario. The path would point to an `emptyDir`. I think I lost that part of the code somehwere along the line to make the PR work.

Reply:
All components now have an emptyDir to store certificates at this location. The advantage is that on crashlooping, we don't have to ask again for a certificate.